source/_integrations/qnap.markdown

```diff
@@ -7,116 +7,28 @@ ha_release: 0.38
-  required: false
-  default: true
-  type: boolean
-timeout:
-  description: How long (in seconds) to wait for a response from the QNAP device before giving up.
-  required: false
-  default: 10
-  type: integer
-username:
-  description: An administration user to connect to the QNAP NAS. This user must be a member of the _administration_ group.
-  required: true
-  type: string
-password:
-  description: The password of the user to connect to the QNAP NAS.
-  required: true
-  type: string
-drives:
-  description: "Array of drives to monitor (ex: `0:1`)."
-  required: false
-  default: all drives
-  type: list
-volumes:
-  description: "Array of volumes to monitor (ex: `DataVol1`)."
-  required: false
-  default: all volumes
-  type: list
-nics:
-  description: "Array of network interfaces to monitor (ex: `eth0`)."
-  required: false
-  default: all NICs
-  type: list
-monitored_conditions:
-  description: Defines the stats to monitor as sensors.
-  required: true
-  type: list
-  keys:
-    status:
-      description: Displays overall system health.
-    system_temp:
-      description: Displays the overall system temperature.
-    cpu_temp:
-      description: Displays the CPU's temperature.
-    cpu_usage:
-      description: Displays the CPU's utilization as a percentage.
-    memory_free:
-      description: Displays the size of available RAM in GB.
-    memory_used:
-      description: Displays the size of used RAM in GB.
-    memory_percent_used:
-      description: Displays the size of used RAM as a percentage of total RAM.
-    network_link_status:
-      description: Displays whether the network interfaces is up (creates a new entry for each interface).
-    network_tx:
-      description: Displays the upload speed of a network interface in MB/s (creates a new entry for each interface).
-    network_rx:
-      description: Displays the download speed of a network interface in MB/s (creates a new entry for each interface).
-    drive_smart_status:
-      description: Displays the S.M.A.R.T. status of the drive (creates a new entry for each drive).
-    drive_temp:
-      description: Displays the temperature of the drive (creates a new entry for each drive).
-    volume_size_free:
-      description: Displays the available space of the volume in GB (creates a new entry for each volume).
-    volume_size_used:
-      description: Displays the used space of the volume in GB (creates a new entry for each volume).
-    volume_percentage_used:
-      description: Displays the used space of the volume as a percentage (creates a new entry for each volume).
-{% endconfiguration %}
+Sensors:
+- system: status, system temperature
+- cpu : usage, temperature
+- memory : usage, free, used, total
+- volume : usage, used
+- folders : usage, used
+- drives: smart status, usage
+- networks uplink : bandwidth, status
 
 ### SSL and Self-signed certificates
```

Comment:
This section needs adjustment as well, as it references to YAML configuration that has been removed.